src/test/java/com/microsoft/azure/spring/data/cosmosdb/core/ReactiveCosmosTemplateIT.java

```diff
@@ -202,18 +199,26 @@ public void testUpsertWithCollectionName() {
     @Test
     public void testDeleteById() {
         cosmosTemplate.insert(TEST_PERSON_4).block();
+        Flux<Person> flux = cosmosTemplate.findAll(Person.class.getSimpleName(), Person.class);
+        StepVerifier.create(flux).expectNextCount(2).verifyComplete();
         final Mono<Void> voidMono = cosmosTemplate.deleteById(Person.class.getSimpleName(), TEST_PERSON_4.getId(),
-                new PartitionKey(TEST_PERSON_4.getId()));
+                PartitionKey.None);
```

Comment:
Does PartitionKey.None mean no partition specified? Is the case with partition key covered?